src/GraphQL/Mutation/MutationType.php

```diff
@@ -397,7 +397,7 @@ public function buildCreateFolderMutation($type, &$config, $context)
         $configuration = $context['configuration'];
         $entities = $configuration->getSpecialEntities();
 
-        if ($entities[$type . "_folder"]["create"]) {
+        if (isset($entities[$type . "_folder"]["create"])) {
```

Comment:
Are you sure? what if the value is FALSE?

Like
                    "asset_folder" => [
                        "id" => "asset_folder",
                        "read" => TRUE,
                        "create" => TRUE,
                        "update" => FALSE,
                        "delete" => TRUE
                    ],

Reply:
Good catch, I'll investigate